### PR TITLE
[Behat] Refactor setup configuration

### PR DESCRIPTION
**Pull request by pamil**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Related tickets | -
| License         | MIT

- [x] Move configuration from `etc/behat/` to `src/Sylius/Behat/Resources/config`
- [x] Divide page definitions into multiple files (instead of a giant one)

### CODE REVIEW

**Comment by pjedrzejewski** [comment]
👍

File: src/Sylius/Behat/Resources/config/services/pages/admin.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        <import resource="admin/product.xml"/>
+        <import resource="admin/product_attribute.xml"/>
+        <import resource="admin/product_option.xml"/>
+        <import resource="admin/promotion.xml"/>
+        <import resource="admin/promotion_coupon.xml"/>
+        <import resource="admin/shipping_method.xml"/>
+        <import resource="admin/tax_category.xml"/>
+        <import resource="admin/tax_rate.xml"/>
+        <import resource="admin/taxon.xml"/>
+        <import resource="admin/zone.xml"/>
```
Comment (michalmarcinkowski) [inline src/Sylius/Behat/Resources/config/services/pages/admin.xml]: Missing blank line before `/>`

Comment (pamil) [inline src/Sylius/Behat/Resources/config/services/pages/admin.xml]: They don't deserve a whole line, a single space will be enough for them :tada:

**Comment by michalmarcinkowski** [comment]
Looks good to me 👍

**Comment by pjedrzejewski** [comment]
Thanks Kamil!

**Pull request merged**